### ISSUE TITLE
Need Only country and state in country.json

### ISSUE THREAD

**Issue by rahmanrezaee** [altafc22/csc_picker#47]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by cvanvlack** [comment]
Agreed that removing the cities would be really helpful. It's a really large file.

Especially because keeping the list of cities up to date seems insanely difficult. I searched for some cities in my geographic area and they weren't on the list, which made me immediately hide the cities dropdown and put in a separate city form field where the user can type in their city directly.

If the cities portion was a separate package that had a dependency on the country/state dropdown then developers would have the option of either!
